### ISSUE TITLE
relion_ctf_toolbox command line input bug

### ISSUE THREAD

**Issue by HamishGBrown** [3dem/relion#802]
Hi, I'm using `relion_ctf_toolbox` to check consistency between a collaborator's code and relion defocus conventions. The `--defV` input seems to be reverting to the default (ie. defV = defU) whenever I input negative values. Here's the commandline input and output:

```
$ relion_ctf_toolbox --simulate ctf.mrc --defV -30                                                                                  + Input values:
 +  kV= 300
 +  Cs= 2.7
 +  Q0= 0.1
 +  defU= 20000
 +  defV= 20000
 +  defAng= 0
 +  phase_shift = 0
 +  angpix= 1
 +  box= 256
 +  use CTF padding? false
 +
 + Done! written: ctf.mrc
$ relion_ctf_toolbox --simulate ctf.mrc --defV 30
 + Input values:
 +  kV= 300
 +  Cs= 2.7
 +  Q0= 0.1
 +  defU= 20000
 +  defV= 30
 +  defAng= 0
 +  phase_shift = 0
 +  angpix= 1
 +  box= 256
 +  use CTF padding? false
 +
 + Done! written: ctf.mrc

```

Also, to confirm, does this flip the contrast of the output CTF to match other relion functions (such as relion_project), so that a particle will appear white against a dark background for underfocus?

Version information:
```
$relion --version
RELION version: 3.1.1-commit-d51f8a
Precision: BASE=double, CUDA-ACC=single
```

**Comment by biochem-fan** [comment]
Thank you very much for your feedback. I fixed this in the commit 01437de.

**Comment by HamishGBrown** [comment]
Thanks for getting onto this so quickly @biochem-fan, I've pulled the latest version, compiled and can confirm that the fix is working as expected on my end.
